build.gradle

```diff
@@ -51,7 +51,7 @@ ext {
   gradleVersion = versions.gradle
   minClientJavaVersion = 11
   minNonClientJavaVersion = 17
-  modulesNeedingJava11 = [":clients", ":generator", ":streams", ":streams:test-utils", ":streams-scala", ":test-common:test-common-util"]
+  modulesNeedingJava11 = [":clients", ":generator", ":streams", ":streams:examples", ":streams:test-utils", ":streams-scala", ":test-common:test-common-util"]
```

Comment:
Nice find, and should we add `examples` module too?

Reply:
Sounds reasonable to me...

\cc @lianetm @kirktrue